src/components/FormFieldComponent.vue

```diff
@@ -137,7 +137,7 @@
 
 <style>
   fieldset.required-field > div > div.form-group > label::after {
-    content: ' *';
+    content: '\a0*';
```

Comment:
I would suggest adding the comment in what '\a0' means and why it is use there.
The comments in the mentioned stackoverflow post mention some issue with the above fix, did you take these into account ?

Reply:
Done.
Taken a closer look at the comments, not relevant here.